### ISSUE TITLE
RESET_BETWEEN_SCENARIOS has stopped working

### ISSUE THREAD

**Issue by ankurgamit** [calabash/run_loop#426]
I'm facing different issues while running tests on calabash-ios 
Everything seems to be working fine until I used @calabash_launcher.reset_simulator  in launch.rb
and I start getting: 

```
 Xcrun timed out after 0.08 seconds executing
  xcrun instruments -s templates
  with a timeout of 30
   (RunLoop::Xcrun::TimeoutError)
  ./features/ios/support/01_launch.rb:28:in `Before'
```
The simulator was launching and was exiting within second.

Then I ran 'calabash-ios sim reset' and tried to run the test again with RESET_BETWEEN_SCENARIOS=1
I start getting this:

```
WARN: deprecated '2.1.0' - No replacement
/Library/Ruby/Gems/2.0.0/gems/run_loop-2.1.0/lib/run_loop/xcode.rb:183:in `version_gte_51?'
/Library/Ruby/Gems/2.0.0/gems/calabash-cucumber-0.18.2/lib/calabash-cucumber/launcher.rb:399:in `default_launch_method'
/Library/Ruby/Gems/2.0.0/gems/calabash-cucumber-0.18.2/lib/calabash-cucumber/launcher.rb:330:in `default_launch_args'
/Library/Ruby/Gems/2.0.0/gems/calabash-cucumber-0.18.2/lib/calabash-cucumber/launcher.rb:434:in `relaunch'

```

Now the simulator launches fine but the app is not getting reset. 

General information:
Xcode version: 7.2.1
run_loop:2.1.0
calabash-ios version: 0.18.2

**Comment by jmoody** [comment]
Thanks for reporting.  I found the problem.

**Comment by jmoody** [comment]
Found a fix - it is bit awkward.  Will be available in 2.1.1.  For now, rollback to 2.0.9.

**Comment by SergiyBeno** [comment]
yes, unfortunately experiencing the same issue:
`/Users/Sergiy/.rvm/gems/ruby-2.2.3/gems/run_loop-2.1.0/lib/run_loop/xcode.rb:183:in `version_gte_51?'
/Users/Sergiy/.rvm/gems/ruby-2.2.3/gems/calabash-cucumber-0.18.2/lib/calabash-cucumber/launcher.rb:399:in `default_launch_method'
/Users/Sergiy/.rvm/gems/ruby-2.2.3/gems/calabash-cucumber-0.18.2/lib/calabash-cucumber/launcher.rb:330:in `default_launch_args'
/Users/Sergiy/.rvm/gems/ruby-2.2.3/gems/calabash-cucumber-0.18.2/lib/calabash-cucumber/launcher.rb:434:in `relaunch'` 
Impatiently awaiting for 2.1.1

**Comment by jmoody** [comment]
@SergiyBeno Please rollback to 2.0.9.  

And the code you pasted seems to have nothing to do with this issue?

**Comment by SergiyBeno** [comment]
Yeah, sorry about that. And yes - downgrade to 2.0.9 fixed the issue.

**Comment by jmoody** [comment]
Fixed.  Will be part of 2.1.1.

**Comment by jmoody** [comment]
Available in 2.1.1.pre1.  Thanks for the report.
